tests/Integration/Behaviour/Features/Context/Domain/AddressFeatureContext.php

```diff
@@ -81,6 +81,9 @@ public function addNewAddressToCustomerWithFollowingDetails(string $customerRefe
     {
         $testCaseData = $table->getRowsHash();
         $customerId = SharedStorage::getStorage()->get($customerReference);
+        if (is_object($customerId)) {
```

Comment:
Don't mix the type of `$customerReference`

Reply:
not sure how to handle this then one feature step expects object and another id.

Reply:
You should refactor to only use object or id